README.md

```diff
@@ -164,7 +166,7 @@ that starts with `create` prefix and provide additional useful functionality:
 
 ## Usage
 
-You can get access to the instance values like with simple PHP `array` behaviour:
+You can get access to the values like with the familiar PHP array indexing:
```

Comment:
Maybe `...like with the familiar PHP array:` without `indexing`? We also could work with assoc arrays.